content/en/observability_pipelines/guide/strategies_for_reducing_log_volume.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Strategies for Reducing Log Volume
+disable_toc: false
+further_reading:
+- link: "/observability_pipelines/set_up_pipelines"
+  tag: "documentation"
+  text: "Set up a pipeline"
+- link: "/observability_pipelines/processors/"
+  tag: "documemtation"
+  text: "Observability Pipelines processors"
+---
+
+## Overview
+
+Observability Pipelines allows you to collect and process logs, as well as determine where you want to route them, all before the data leaves your on-premises or cloud environment.
+
+The value of your logs is not all equal. For example, error logs are often more useful than info logs when troubleshooting live operations. Logs from production environments are also more important than logs from non-production environments. Therefore, routing all of your logs to an indexed solution can end up diluting the overall value of your data and cause you to exceed your budget.
+
+The following situations can also unnecessarily increase your log volume and cost:
+
+- An application team inadvertently turns on debug logging and a new build triggers a flood of errors.
+- Teams attempt to add as much performance and metric data into the logs because it's easier than separating out the data.
```

Comment:
```suggestion
- Teams attempt to add as much performance and metric data into the logs because it seems easier than implementing other telemetry solutions.
```